### ISSUE TITLE
CPU PERCENTAGE GONE

### ISSUE THREAD

**Issue by gnolooo** [corecoding/Vitals#256]
### Has this issue been covered in the Wiki?

- [X] I have read the Wiki and my issue is not covered.

### Is there an existing issue reported already?

- [x] I have searched the existing issues, it does not exist.

### What is the unexpected behavior?

After the latest update, I can't set as a pinnable icon the average CPU percentage usage. I can only pin the Bogomips value, the boost, the cache and the single cores (unlike for example in RAM, where the average percentage is present).

This is what I see:
![Schermata del 2022-05-07 12-40-41](https://user-images.githubusercontent.com/76843243/167250830-9aab3793-e784-426f-9abb-6a9ce34d6a82.png)

### Steps to reproduce the unexpected behavior.

1. Go to top panel
2. Click the CPU section

### What distribution and version of Linux are you using?

Arch Linux

### What version of Gnome are you using?

42

**Comment by corecoding** [comment]
When you say you can't "set as pinnable" what do you mean? When you click Media under Processor, what exactly happens?

**Comment by gnolooo** [comment]
When I say "set as pinnable" I mean clickable to pin it on the dash. There's no "average CPU usage" section (except for the sub-menu one) that I can click to pin to the dash, as I can do for example with RAM instead.
Basically I would like to pin this voice (which is average CPU usage) on the dash:
![2022-05-07_18-45](https://user-images.githubusercontent.com/76843243/167263861-46b03461-0c93-4215-ba6d-adf53568ff85.png)
but I can't do so since the voice is missing in the submenu.
I'm pretty sure this is a bug because it was there until last update, and I don't think you remove it. But I've tried uninstalling it, clearing cache etc, and nothing worked.

For reference, this is what I used to configure Vitals like (sorry for bad quality, it's an old screenshot):
![image](https://user-images.githubusercontent.com/76843243/167263998-5fb940ed-3110-4240-be01-015b5b1715e7.png)

P.S.: the CPU icon is different just because I changed it, but afaik that shouldn't affect the extension behavior itself.
